routes/api/bookin.js

```diff
@@ -95,7 +95,26 @@ router.get("/history", passport.authenticate("jwt", { session: false }), (req, r
                             });
 
                             // Check if we've finished packing ALL the history objects
-                            if (historyPack.length === bookings.length) return res.status(200).send(historyPack);
+                            if (historyPack.length === bookings.length) {
```

Comment:
what is this?

Reply:
I don't know why this happened... I fixed it back